samples/net/coap_client/src/main.c

```diff
@@ -21,30 +22,26 @@ LOG_MODULE_REGISTER(coap_client_sample, CONFIG_COAP_CLIENT_SAMPLE_LOG_LEVEL);
 #define L4_EVENT_MASK (NET_EVENT_L4_CONNECTED | NET_EVENT_L4_DISCONNECTED)
 #define CONN_LAYER_EVENT_MASK (NET_EVENT_CONN_IF_FATAL_ERROR)
 
-#define APP_COAP_SEND_INTERVAL_MS 5000
-#define APP_COAP_MAX_MSG_LEN 1280
+#define APP_COAP_SEND_INTERVAL_SECONDS 5
```

Comment:
Making this option configurable would be nice.

Reply:
I agree on that, but I had some issues when trying to make it configurable, so I went back to how it was previously.. I can test it again when I am back at the office tomorrow.